client/src/App.js

```diff
@@ -4,27 +4,35 @@ import Header from './Components/Header';
 import DropDown from './Components/DropDown';
 import SearchButton from './Components/Button';
 import ResultBox from './Components/ResultBox';
+import { products } from './Components/productData';
+import ProfilePicture from '../src/Components/Picture/ProfilePicture.png';
 
 function App() {
-  const [product, setProduct] = useState();
   return (
     <body className="body">
       <form>
         <Header />
         <h1 className="question">What would You like to give back?</h1>
-        <DropDown />
+        <article>
+          <img
+            class="Profile Picture"
+            src={ProfilePicture}
+            alt="better be a pic"
+            width="30%"
+            height="30%"
+            justify-content="center"
+            display="flex"
+          />
+        </article>
+        {/* DropDown werden die Daten von productData.js übergeben: App.js -> DropDown.js */}
```

Comment:
please delete unnecessary code here, especially in { }. If your code is super long and hard to understand, you may leave a comment like this. In all other cases, let the code speak for itself :)